pkg/subscriptions/subscriptions-register.jsx

```diff
@@ -0,0 +1,239 @@
+            this.props.onChange('proxy', !this.props.proxy);
+            // don't actually navigate
+            event.preventDefault();
+        },
+        render: function() {
+            var customURL;
+            if (this.props.url == 'custom') {
+                customURL = (
+                    <input id="subscription-register-url-custom" className="form-control" type="text"
+                     value={this.props.serverUrl} onChange={this.props.onChange.bind(this, 'serverUrl')} />
+                );
+            }
+            var proxy;
+            if (this.props.proxy) {
+                proxy = [
+                    <br />,
+                    <table className="form-group-ct">
+                        <tbody>
+                            <tr>
+                                <td>
+                                    <label className="control-label form-tr-ct-title" for="subscription-proxy-server">
+                                        {_("Server")}
+                                    </label>
+                                </td>
+                                <td><input className="form-control" id="subscription-proxy-server" type="text"
+                                           placeholder="hostname:port" value={this.props.proxyServer}
+                                           onChange={this.props.onChange.bind(this, 'proxyServer')}/>
+                                </td>
+                            </tr>
+
+                            <tr>
+                                <td>
+                                    <label className="control-label form-tr-ct-title" for="subscription-proxy-user">
+                                        {_("User")}
+                                    </label>
+                                </td>
+                                <td><input className="form-control" id="subscription-proxy-user" type="text"
+                                           value={this.props.proxyUser}
+                                           onChange={this.props.onChange.bind(this, 'proxyUser')}/>
+                                </td>
+                            </tr>
+
+                            <tr>
+                                <td>
+                                    <label className="control-label form-tr-ct-title" for="subscription-proxy-password">
+                                        {_("Password")}
+                                    </label>
+                                </td>
+                                <td><input className="form-control" id="subscription-proxy-password" type="password"
+                                           value={this.props.proxyPassword}
+                                           onChange={this.props.onChange.bind(this, 'proxyPassword')}/>
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
+                ];
+      /*
```

Comment:
> Forgot to remove this comment?

it was left in place because the proxy design is off and needs fixing, but removed it now